examples/contexts/README.md

```diff
@@ -0,0 +1,68 @@
+# Using Contexts to share configuration & state
+
+A Panel component can be configured with Contexts to share configuration & state in its DOM tree instead of manually passing them into each component as props/attrs.
```

Comment:
```suggestion
A Panel component can be configured with Contexts to share configuration and state in its DOM tree instead of manually passing them into each component as props/attrs.
```